Modules/Filtering/ImageGrid/include/itkCropImageFilter.h

```diff
@@ -111,6 +111,9 @@ class ITK_TEMPLATE_EXPORT CropImageFilter : public ExtractImageFilter<TInputImag
   void
   GenerateOutputInformation() override;
 
+  void
+  VerifyInputInformation() ITKv5_CONST override;
```

Comment:
`ITKv5_CONST` is meant for user code which need to build against both v4 and v5. Here we should use plain `const`, I believe.

Reply:
The const-ness was added in ITKv5 in the base class and it's toggled with the compatibility flags. So it's needed.
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkProcessObject.h#L789

Reply:
Human memory is faulty 😄